crates/bitwarden-fido/src/lib.rs

```diff
@@ -102,14 +105,21 @@ impl TryFrom<CipherViewContainer> for Passkey {
 fn try_from_credential_full_view(value: Fido2CredentialFullView) -> Result<Passkey, Fido2Error> {
     let counter: u32 = value.counter.parse().expect("Invalid counter");
     let counter = (counter != 0).then_some(counter);
+    log::info!("Converting Credential to Passkey: {:?}", value);
+    let key_value = URL_SAFE_NO_PAD.decode(value.key_value)?;
+    let user_handle = value
+        .user_handle
+        .map(|u| URL_SAFE_NO_PAD.decode(u))
+        .transpose()?;
+    log::info!("Key value: {:?}", key_value);
```

Comment:
I think we should be careful about logging key values, or at least use `log::debug!` or `log::trace!`, which should not print on release builds by default.

Reply:
Oooooooooooff I completely missed this, it was a debug log and I definitely did not intend on this being logged. Thanks for the save!

Reply:
Removed!